include/Beman/Execution26/detail/sender_in.hpp

```diff
@@ -0,0 +1,34 @@
+// include/Beman/Execution26/detail/sender_in.hpp                     -*-C++-*-
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef INCLUDED_BEMAN_EXECUTION26_DETAIL_SENDER_IN
+#define INCLUDED_BEMAN_EXECUTION26_DETAIL_SENDER_IN
+
+#include <Beman/Execution26/detail/empty_env.hpp>
+#include <Beman/Execution26/detail/get_completion_signatures.hpp>
+#include <Beman/Execution26/detail/queryable.hpp>
+#include <Beman/Execution26/detail/sender.hpp>
+#include <Beman/Execution26/detail/valid_completion_signatures.hpp>
+
+// ----------------------------------------------------------------------------
+
+namespace Beman::Execution26
```

Comment:
::Detail?  the file is under the directory?

Reply:
This comment is interesting: `sender_in` is supposed to be in `std::execution`. However, the file `sender_in.hpp` isn't anything the standard defines: the header is `<execution>` which is modelled by `include/Beman/Execution26/execution.hpp` which would eventually include `sender_in.hpp`.

I can't cope with creating just one header [manually], i.e., I'm organising my implementation mostly into one component per header. The header name is an implementation-specific detail, the name within is (in this case) a standard name. As a result, the namespace doesn't quite line up with the path.

Maybe the proper way is to actually define `sender_in` in the `Detail` namespace and get it into the standard namespace using a suitable alias (and, obviously, do so far other names, too). That sounds like a reasonable thing to do and I'll make a pass through the code to change it accordingly.